CHANGELOG.rst

```diff
@@ -6,6 +6,130 @@
 Synapse Changelog
 *****************
 
+v2.156.0 - 2023-12-08
+=====================
+
+Model Changes
+-------------
+- Updates to the ``infotech``, ``ou``,  and ``risk`` models.
+  (`#3436 <https://github.com/vertexproject/synapse/pull/3436>`_)
+  (`#3438 <https://github.com/vertexproject/synapse/pull/3438>`_)
+  (`#3446 <https://github.com/vertexproject/synapse/pull/3447>`_)
+  (`#3447 <https://github.com/vertexproject/synapse/pull/3447>`_)
+
+  **New Properties**
+
+  ``it:av:scan:result``
+    The form had the following properties added to it:
+
+      ``target:ipv4``
+        The IPv4 that was scanned to produce the result.
+
+      ``target:ipv6``
+        The IPv6 that was scanned to produce the result.
+
+  ``ou:campaign``
+    The form had the following property added to it:
+
+    ``mitre:attack:campaign``
+      A mapping to a Mitre ATT&CK campaign if applicable.
+
+  ``risk:vuln``
+    The form had the following property added to it:
+
+    ``id``
+      An identifier for the vulnerability.
+
+  **New Forms**
+
+  ``it:mitre:attack:campaign``
+    A Mitre ATT&CK Campaign ID.
```

Comment:
```suggestion
    A Mitre ATT&CK Campaign.
```

Reply:
They really are campaign IDs though :shrug:

Reply:
That's fair. Never mind then :)

Reply:
This is the form ``doc``. Should the form doc be updated?